### DISCUSSION TITLE
Reconsider the SEQ (sequence) property

### DISCUSSION THREAD

**Discussion by dse-copsfs** [usebruno/bruno#2149] [Ideas]
## Workflow

1. Add a new request in the existing folder of 10 requests at the second position.
1. All existing requests after the second position are modified because the `seq` property is incremented by one.
![image](https://github.com/usebruno/bruno/assets/92361097/747b6e7a-d745-4ef9-86b7-fbdcd32c4deb)

## Idea

I think it's against the nice GIT-friendly approach – the change of one added file should not modify the rest of the existing files. Please reconsider the usage of the `seq` property.

What about to store sequence order info in separate file on the level of the concrete folder?

**Comment by end3rbyte** [comment]
I agree!

**Comment by sean-bp** [comment]
Agreed. Completely muddies up the commits IMO!

**Comment by in4cookies** [comment]
Hi, is there an enhancement issue opened (I cannot find it) for this or is only a discussion?
